crates/edr_provider/src/config.rs

```diff
@@ -58,10 +58,10 @@ pub struct MiningConfig {
 /// Configuration for the provider
 #[derive(Debug, Clone, Deserialize, Serialize)]
 pub struct ProviderConfig {
-    pub allow_blocks_with_same_timestamp: bool,
-    pub allow_unlimited_contract_size: bool,
+    pub allow_blocks_with_same_timestamp: bool, // x
+    pub allow_unlimited_contract_size: bool,    // x
     pub accounts: Vec<AccountConfig>,
-    /// Whether to return an `Err` when `eth_call` fails
+    /// Whether to return an `Err` when `eth_calll fails
```

Comment:
Seemingly erroneously introduced changes

Reply:
Thanks fixed in [daa0615](https://github.com/NomicFoundation/hardhat/pull/4978/commits/daa061553a1a33cecfe7e55f4fe978fc1e8980fc)